firestore-translate-text/functions/src/logs/index.ts

```diff
@@ -14,99 +14,99 @@
  * limitations under the License.
  */
 
-import config from "../config";
+import { logger } from "firebase-functions";
 import { messages } from "./messages";
 
 export const complete = () => {
-  console.log(messages.complete());
+  logger.log(messages.complete());
 };
 
 export const documentCreatedNoInput = () => {
-  console.log(messages.documentCreatedNoInput());
+  logger.log(messages.documentCreatedNoInput());
 };
 
 export const documentCreatedWithInput = () => {
-  console.log(messages.documentCreatedWithInput());
+  logger.log(messages.documentCreatedWithInput());
 };
 
 export const documentDeleted = () => {
-  console.log(messages.documentDeleted());
+  logger.log(messages.documentDeleted());
 };
 
 export const documentUpdatedChangedInput = () => {
-  console.log(messages.documentUpdatedChangedInput());
+  logger.log(messages.documentUpdatedChangedInput());
 };
 
 export const documentUpdatedDeletedInput = () => {
-  console.log(messages.documentUpdatedDeletedInput());
+  logger.log(messages.documentUpdatedDeletedInput());
 };
 
 export const documentUpdatedNoInput = () => {
-  console.log(messages.documentUpdatedNoInput());
+  logger.log(messages.documentUpdatedNoInput());
 };
 
 export const documentUpdatedUnchangedInput = () => {
-  console.log(messages.documentUpdatedUnchangedInput());
+  logger.log(messages.documentUpdatedUnchangedInput());
 };
 
 export const error = (err: Error) => {
-  console.error(...messages.error(err));
+  logger.error(...messages.error(err));
 };
 
 export const fieldNamesNotDifferent = () => {
-  console.error(messages.fieldNamesNotDifferent());
+  logger.error(messages.fieldNamesNotDifferent());
 };
 
-export const init = () => {
-  console.log(...messages.init(config));
+export const init = (config) => {
+  logger.log(...messages.init(config));
 };
 
 export const inputFieldNameIsOutputPath = () => {
-  console.error(messages.inputFieldNameIsOutputPath());
+  logger.error(messages.inputFieldNameIsOutputPath());
 };
 
 export const start = () => {
-  console.log(...messages.start(config));
+  logger.log(messages.start());
```

Comment:
Looks like we lost the config printout here - is that intentional? If not, it could be added as the second argument to `logger.log`

Reply:
Looks like `config` was moved to the `init` logger

Reply:
I updated the code to previous behaviour so the config is printed out for each execution. I don't think it's my call to change it.